draft-ietf-taps-interface.md

```diff
@@ -1692,13 +1692,12 @@ Connection Properties represent the configuration and state of the selected
 Protocol Stack(s) backing a Connection. These Connection Properties may be
 Generic, applying regardless of transport protocol, or Specific, applicable to a
 single implementation of a single transport protocol stack. Generic Connection
-Properties are defined in {{connection-props}} below. Specific Protocol
-Properties are defined in a transport- and implementation-specific way, and MUST
-NOT be assumed to apply across different protocols. Attempts to set Specific
-Protocol Properties on a protocol stack not containing that specific protocol
-are simply ignored, and do not raise an error; however, too much reliance by an
-application on Specific Protocol Properties can significantly reduce the
-flexibility of a transport services implementation.
+Properties are defined in {{connection-props}} below. 
+
+Specific Protocol Properties are defined in a transport- and 
```

Comment:
```suggestion
Protocol Specific Properties are defined in a transport- and 
```

Reply:
This terminology is what's already used in the doc, so I say to keep it.

Reply:
Yes, in the current version, we have three times "Protocol Specific Properties" and three times "Specific Protocol Properties" meaning the same thing. We should really unify on one term.

Reply:
Oh yes, absolutely, it should be Protocol Specific Properties everywhere  (actually I wonder if this wouldn't be even better with a dash?  I.e., "Protocol-specific Properties"?).

To me, "Specific Protocol Properties" doesn't read like it would even mean the same thing, it reads to me as "specific ones as opposed to very vague ones"  (well, as opposed to "generic" ones ... so that kinda works, but it's less clear that these are specific *to* a protocol, IMO).